libevmasm/AssemblyItem.cpp

```diff
@@ -102,9 +102,13 @@ std::pair<std::string, std::string> AssemblyItem::nameAndData(langutil::EVMVersi
 		return {"PUSH data", toStringInHex(data())};
 	case VerbatimBytecode:
 		return {"VERBATIM", util::toHex(verbatimData())};
-	default:
-		assertThrow(false, InvalidOpcode, "");
+	case AuxDataLoadN:
+		return {"AUXDATALOADN", util::toString(data())};
+	case UndefinedItem:
+		solThrow(InvalidOpcode, "");
```

Comment:
If it's not an assert, it needs a message, because it may reach the user in normal usage.

But to me it does look like an assert. `UndefinedItems` seems to be just an intermediate placeholder value that should never end up in the assembly.
```suggestion
		solAssert(false);
```

Same in `bytesRequired()` below.